notebooks/community/vizier/conversions_vertex_vizier_and_open_source_vizier.ipynb

```diff
@@ -0,0 +1,621 @@
+        "    </a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/vizier/gapic-vizier-multi-objective-optimization.ipynb\">\n",
+        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
+        "      View on GitHub\n",
+        "    </a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/workbench/deploy-notebook?download_url=https://raw.githubusercontent.com/GoogleCloudPlatform/vertex-ai-samples/main/notebooks/official/vizier/gapic-vizier-multi-objective-optimization.ipynb\">\n",
+        "        <img src=\"https://lh3.googleusercontent.com/UiNooY4LUgW_oTvpsNhPpQzsstV5W8F7rYgxgGBD85cWJoLmrOzhVs_ksK_vgx40SHs7jCqkTkCk=e14-rj-sc0xffffff-h130-w32\" alt=\"Vertex AI logo\">\n",
+        "      Open in Vertex AI Workbench\n",
+        "    </a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "b397c59391b1"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "This tutorial demonstrates [Vertex AI Vizier](https://cloud.google.com/vertex-ai/docs/vizier/overview) and how to convert the code to [Open Source(OSS) Vizier](https://oss-vizier.readthedocs.io/). OSS Vizier is a Python-based service for blackbox optimization and research. It allows a user to setup an OSS Vizier Server, which can host blackbox optimization algorithms to tune the objective functions, and to define abstractions and utilities for implementing new optimization algorithms. \n",
+        "\n",
+        "If you are an existing OSS Vizier user, the instructions show you how to migrate the code to Vertex Vizier SDK. If you are a user of Vertex Vizier, the instructions show you how to modify the code to adapt to OSS Vizier."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "AksIKBzZ-nre"
+      },
+      "source": [
+        "### Objective\n",
+        "\n",
+        "In this tutorial, you learn how to use `Vertex AI Vizier` to optimize a multi-objective study and convert the code to OSS Vizier.\n",
+        "\n",
+        "The goal is to __`minimize`__ the objective metric:\n",
+        "   ```\n",
+        "   y1 = r*sin(theta)\n",
+        "   ```\n",
+        "\n",
+        "and simultaneously __`maximize`__ the objective metric:\n",
+        "   ```\n",
+        "   y2 = r*cos(theta)\n",
+        "   ```\n",
+        "\n",
+        "that you will evaluate over the parameter space:\n",
+        "\n",
+        "   - __`r`__ in [0,1],\n",
+        "\n",
+        "   - __`theta`__ in [0, pi/2]\n",
+        "\n",
+        "### Costs\n",
+        "\n",
+        "This tutorial uses billable components of Google Cloud:\n",
+        "\n",
+        "* Vertex AI\n",
```

Comment:
Should this also include Cloud Storage and Compute Engine?

Reply:
In the colab, we didn't create training job and store anything.